### DISCUSSION TITLE
Question About base("com.yasirkula.unity.NativeGalleryMediaReceiver")

### DISCUSSION THREAD

**Discussion by RusticCupsJava** [yasirkula/UnityNativeGallery#233] [Q&A]
Hello, thank you so much for making this plugin, it's so cool and so clean to follow! I am newer to programming and I am not able to understand what the following part means in one of the classes:

base("com.yasirkula.unity.NativeGalleryMediaReceiver")

For the class public NGMediaReceiveCallbackAndroid

**Comment by yasirkula** [comment]
It lets us create an instance of this native Java class on Android-side: https://github.com/yasirkula/UnityNativeGallery/blob/master/.github/AAR%20Source%20(Android)/java/com/yasirkula/unity/NativeGalleryMediaReceiver.java

**Reply by RusticCupsJava** [reply]
Thank you so much! So does that mean when I use the plugin, I use your class from the package I downloaded into Unity? Or does it get the class from your github repo?

**Reply by yasirkula** [reply]
This class is embedded in NativeGallery.aar. The link I've posted is there for reference only.
